Traceback (most recent call last):
  File "/data/gc/anaconda3/envs/lm/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/data/gc/anaconda3/envs/lm/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/data3/gc/SeLM_v2/src/gpt.py", line 485, in <module>
    train_cte(train_cache_ckpt, valid_cache_ckpt, gpt_ckpt, N_train, N_valid)
  File "/data/gc/anaconda3/envs/lm/lib/python3.10/site-packages/torch/utils/_contextlib.py", line 116, in decorate_context
    return func(*args, **kwargs)
  File "/data3/gc/SeLM_v2/src/gpt.py", line 453, in train_cte
    cte.train_all(
  File "/data3/gc/SeLM_v2/src/gettime.py", line 118, in wrapper
    return fn(*args, **kwargs)
  File "/data3/gc/SeLM_v2/src/cte.py", line 627, in train_all
    self.converge  = loss_strategy['train_converge'] is not None and (self.cur_epoch >= loss_strategy['train_converge'])
NameError: name 'loss_strategy' is not defined. Did you mean: 'get_strategy'?
